src/pe_source/cybersixgill.py

```diff
@@ -0,0 +1,407 @@
+        if "topCVEs" in method_list:
+            if self.get_topCVEs(source_uid) == 1:
+                failed.append("Top CVEs")
+
+        list = ""
+        for pe_org in pe_orgs:
+            list = list + pe_org["cyhy_db_name"] + ","
+        print(list)
+        for pe_org in pe_orgs:
+            org_id = pe_org["cyhy_db_name"]
+            pe_org_uid = pe_org["org_uid"]
+            # Only run on specified orgs
+            if org_id in orgs_list or orgs_list == "all":
+                count += 1
+                # Get sixgill_org_id associated with the PE org
+                try:
+                    sixgill_org_id = sixgill_orgs[org_id][5]
+                except KeyError as err:
+                    LOGGER.error("PE org is not listed in Cybersixgill.")
+                    print(err, file=sys.stderr)
+                    failed.append("%s not in sixgill" % org_id)
+                    continue
+
+                # Run alerts
+                if "alerts" in method_list:
+                    if (
+                        self.get_alerts(
+                            org_id,
+                            sixgill_org_id,
+                            pe_org_uid,
+                            source_uid,
+                            soc_med_included,
+                        )
+                        == 1
+                    ):
+                        failed.append("%s alerts" % org_id)
+                # Run mentions
+                if "mentions" in method_list:
+                    if (
+                        self.get_mentions(
+                            org_id,
+                            sixgill_org_id,
+                            pe_org_uid,
+                            source_uid,
+                            soc_med_included,
+                        )
+                        == 1
+                    ):
+                        failed.append("%s mentions" % org_id)
+                # Run credentials
+                if "credentials" in method_list:
+                    if (
+                        self.get_credentials(
+                            org_id, sixgill_org_id, pe_org_uid, source_uid
+                        )
+                        == 1
+                    ):
+                        failed.append("%s credentials" % org_id)
+        if len(failed) > 0:
+            LOGGER.error("Failures: %s", failed)
+
+    def get_alerts(
+        self, org_id, sixgill_org_id, pe_org_uid, source_uid, soc_med_included
+    ):
+        """Get alerts."""
+        LOGGER.info("Fetching alert data for %s.", org_id)
+        soc_med_platforms = [
+            "twitter",
+            "Twitter",
+            "reddit",
+            "Reddit",
+            "Parler",
+            "parler",
+            "linkedin",
+            "Linkedin",
+            "discord",
+            "forum_discord",
+            "raddle",
+            "telegram",
+            "jabber",
+            "ICQ",
+            "icq",
+            "mastodon",
```

Comment:
For ease of maintenance, can this list be sorted alphabetically?
```suggestion
            "discord",
            "forum_discord",
            "icq",
            "ICQ",
            "jabber",
            "linkedin",
            "Linkedin",
            "mastodon",
            "parler",
            "Parler",
            "raddle",
            "reddit",
            "Reddit",
            "telegram",
            "twitter",
            "Twitter",
```